mygpoclient/api_test.py

```diff
@@ -222,14 +222,14 @@ def setUp(self):
     def set_http_response_value(self, value):
         self.fake_client.response_value = value
 
-    def assert_http_request_count(self, count):
-        self.assertEquals(len(self.fake_client.requests), count)
+    def assertTruehttp_request_count(self, count):
```

Comment:
It would read better as `assertEqual_http_request_count` since it does not compare a boolean value, and the underscore separates the lowercase letters.

Reply:
That was in error, corrected.